modules/virt-about-eus-updates.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * virt/upgrading-virt.adoc
+
+:_content-type: CONCEPT
+[id="virt-about-eus-updates_{context}"]
+= About EUS-to-EUS updates
+
+Every even-numbered minor version of {product-title}, including 4.10 and 4.12, is an Extended Update Support (EUS) version. However, because Kubernetes design mandates serial minor version updates, you cannot directly update from one EUS version to the next. You must first update to the odd-numbered minor version before you can proceed to the target EUS version.
+
+When performing an EUS-to-EUS update, you must:
+
+* Pause worker nodes' machine config pools before you start an EUS-to-EUS update so that the workers are not rebooted twice. 
+
+* Disable automatic workload updates before you begin the update process. This is to prevent {VirtProductName} from migrating or evicting your virtual machines (VMs) when you update to the odd-numbered minor version.
```

Comment:
How about instead of `when you update to the odd-numbered minor version`, we state something like `till you upgrade to your target EUS version`

Reply:
Love this suggestion, thanks!